code/game/cas_manager/datums/cas_fire_mission.dm

```diff
@@ -172,3 +172,41 @@
 		sleep(step_delay)
 	if(envelope)
 		envelope.change_current_loc(null)
+
+// Used only in the simulator room for testing firemissions. IMO it was ok to copy code here.
+/datum/cas_fire_mission/proc/simulate_execute_firemission(obj/structure/machinery/computer/dropship_weapons/linked_console, turf/initial_turf, direction = NORTH, steps = 12, step_delay = 3, datum/cas_fire_envelope/envelope = null)
+	if(initial_turf == null)
+		return -1
```

Comment:
this would probably be best as a define if it does anything

https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#no-magic-numbers-or-strings